EG23.tex

```diff
@@ -157,16 +161,29 @@ \subsection{Reactors}
 \begin{table}[h!]
     \centering
     \begin{tabularx}{350pt}{lXXX}
-    \hline
-    Core Properties       &	LWR     &	SFR 1   &	SFR 2     \\
-    \hline
-    Rated Power, [MWe]    &	1000		&	400     &	400       \\
-    Thermal Efficiency    &	N.A.$^1$	&	0.4     &	0.4       \\
-    Capacity Factor       &	0.90		&	0.90		&	0.90      \\
-    Number of batches     &	6       &	5       &	3         \\
-    Cycle length, [month] &	9       &	13      &	18        \\
-    Core Inventory, [tHM] &	6$\times$14.784 & 5$\times$7.524 & 3$\times$11.466  \\
-    \hline
+      \hline
+      Core Properties       &  LWR             &  SFR 1           &  SFR 2            \\
+      \hline
+      Rated Power, [MWe]    &  1000            &  400             &  400              \\
+      Thermal Efficiency    &  N.A.$^1$        &  0.4             &  0.4              \\
+      Capacity Factor       &  0.90            &  0.90            &  0.90             \\
+                                                                                      \\
+      \multicolumn{4}{X}{Specifications}                                              \\ 
+      \hline
+      \hline
+      Number of batches     &  3               &  5.44            &  4.44             \\
+      Cycle length, [month] &  18              &  1               &  1                \\
+      Core Inventory, [tHM] &  3$\times$29.565 &  37.62           &  34.40            \\
+                                                                                      \\
+      \multicolumn{4}{l}{Cyclus inplementation}                                       \\ 
+      \hline
+      \hline
+      Number of batches     &  6               &  5               &  3                \\
```

Comment:
Do you know why we switched to 6 batches?  It seems that we could have done this as 3 batches, right?